.devcontainer/bazel-base/Dockerfile

```diff
@@ -17,69 +17,69 @@ RUN echo "Install general purpose packages" && \
     apt-get update && \
     apt-get upgrade -y && \
     apt-get install -y --no-install-recommends \
-        apt-transport-https \
```

Comment:
Can we keep the formatting as it is in this file? The extra spaces are here because these are packages for the `apt-get install` and it is more readable this way.